classes/class-twenty-twenty-one-customize.php

```diff
@@ -81,6 +81,41 @@ public function register( $wp_customize ) {
 				)
 			);
 
+			/**
+			 * Add excerpt or full text selector to customizer
+			 */
+			$wp_customize->add_section(
+				'theme_settings',
+				array(
+					'title'    => esc_html__( 'Theme settings', 'twentytwentyone' ),
+					'priority' => 30,
+				)
+			);
+
+			$wp_customize->add_setting(
+				'display_excerpt_or_fullpost',
```

Comment:
Shouldn't this be `display_excerpt_or_full_post`?

Reply:
Thank you @ZebulanStanphill for the review :+1: 
Fixed in https://github.com/WordPress/twentytwentyone/pull/379